mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -316,6 +316,28 @@ def _get_run_relative_artifact_path_for_upload(self, src_file_path, dst_artifact
             run_relative_artifact_path = self.run_relative_artifact_repo_root_path
         return run_relative_artifact_path
 
+    def _set_databricks_host_creds_to_credential_context(self):
+        db_creds = get_databricks_host_creds(self.databricks_profile_uri)
+        dbutils = _get_dbutils()
+        dbutils.endpoint.putMlflowProperties(
+            db_creds.host,
+            db_creds.ignore_tls_verification,
+            db_creds.token,
+            db_creds.username,
+            db_creds.password,
+        )
+
+    def _clear_credential_context(self):
+        dbutils = _get_dbutils()
+        dbutils.endpoint.clearMlflowProperties()
+
+    def _get_mlflowdbfs_path(self, artifact_path):
+        if artifact_path.startswith("/"):
+            raise MlflowException(
+                "artifact_path should be relative, found: {}".format(artifact_path)
+            )
+        return "mlflowdbfs:///artifacts?run_id={}&path=/{}".format(self.run_id, artifact_path)
```

Comment:
I don't think these methods should be added to `DatabricksArtifactRepository`, since they're orthogonal (`DatabricksArtifactRepository` is not used for file uploads / downloads when mlflowdbfs is available). Can we put them in `mlflow.utils.databricks_utils` instead?